src/chocolatey.resources/helpers/functions/Get-WmiObject.ps1

```diff
@@ -0,0 +1,15 @@
+﻿## Get-WmiObject
+##############################################################################################################
+## Wrapper function. Replaces "Win32_Product" class with Get-Installer helper.
+##############################################################################################################
+
+function Get-WmiObject {
```

Comment:
Wrapper function may take any amount of arguments. It's just passes them to original function if there are no "Win32_Product". When no arguments are passed, the original function asks a user to input at least "class" argument.